thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+add_subdirectory(xbyak)
+
+openvino_developer_export_targets(COMPONENT openvino_common TARGETS xbyak)
```

Comment:
is it used by 3rd party? As we discussed, I think it can be removed

Reply:
It seems yes. I tried to remove xbyak exporting but it works only on local workstation. In CI many tests weren't built because xbyak library wasn't found.